.github/workflows/phpmetrics.yml

```diff
@@ -0,0 +1,86 @@
+name: Deploy PhpMetrics to GitHub Pages
+
+on:
+  push:
```

Comment:
@maluueu Shouldn't this be set to tag?

Reply:
We can either leave it at push so PhpMetrics always represents the latest changes (e.g. master branch) or change it to tag, so it always is coupled to the latest released tag.

Reply:
I prefer latest tag